django/contrib/auth/management/__init__.py

```diff
@@ -108,6 +109,90 @@ def create_permissions(
             print("Adding permission '%s'" % perm)
 
 
+class RenamePermission(migrations.RunPython):
+    def __init__(self, app_label, old_model, new_model):
+        self.app_label = app_label
+        self.old_model = old_model
+        self.new_model = new_model
+        super(RenamePermission, self).__init__(
+            self.rename_forward, self.rename_backward
+        )
+
+    def _rename(self, apps, schema_editor, old_model, new_model):
+        ContentType = apps.get_model("contenttypes", "ContentType")
+        from django.contrib.auth.models import Permission  # real live model import
```

Comment:
Can you explain why this shouldn't be `Permission = apps.get_model("auth", "Permission")`?
I see there are some test failures but the direct import feels wrong

Reply:
I think this is the best option we have :+1: